mirrord/cli/src/config.rs

```diff
@@ -147,3 +150,26 @@ pub(super) struct LoginArgs {
     #[clap(long)]
     pub no_open: bool,
 }
+
+#[derive(Args, Debug)]
+pub(super) struct OperatorArgs {
+    #[clap(subcommand)]
+    pub command: OperatorCommand,
+}
+
+#[derive(Subcommand, Debug)]
+pub(super) enum OperatorCommand {
+    /// This will install the operator, which requires a seat based license to be used.
+    ///
+    /// NOTE: You don't need to install the operator to use open source mirrord features.
+    #[clap(override_usage = "mirrord operator setup [OPTIONS] | kubectl apply -f -")]
+    Setup {
+        /// Output to kubernetes specs to file instead of stdout and piping to kubectl
+        #[clap(short, long, value_parser)]
+        file: Option<PathBuf>,
+
+        /// Set namespace to setup operator in
```

Comment:
Add a comment that this doesn't limit the namespaces the operator will be able to work in (IIRC?)